### PR TITLE
[MM-48572] : Replace --center-channel-text css variable with --center-channel-color

### PR DESCRIPTION
**Pull request by AshishDhama**
#### Summary
 Replace --center-channel-text css variable with --center-channel-color

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-48572

#### Related Pull Requests
NA

#### Screenshots
NA

#### Release Note
```release-note
*  Replace --center-channel-text css variable with --center-channel-color
```

### CODE REVIEW

**Comment by AshishDhama** [comment]
cc// @esethna @matthewbirtch

**Review by JulienTant — APPROVED** [review]
LGTM! Thanks for the simplification!

File: components/pricing_modal/pricing_modal.scss [hunk 1]
```diff
@@ -298,7 +298,7 @@
                     position: relative;
                     height: auto;
                     padding: 37px 24px 24px 24px;
-                    background-color: var(--title-color-indigo-500);
+                    background-color: var(--sidebar-bg);
```
Comment (julmondragon) [inline components/pricing_modal/pricing_modal.scss:301]: Wondering about this change, as the design specifies the background color not changing with themes:
https://www.figma.com/file/rWB5bbi3DvpmVwVMc2xfOH/Update-Plans?node-id=0%3A1&t=EfrG35qAEXy6HlnJ-0

Screen Shot 2022-11-23 at 11 08 13 AM

cc @AGMETEOR @annelieseklein

**Review by julmondragon — APPROVED** [review]
LGTM! Thanks @AshishDhama

File: components/pricing_modal/pricing_modal.scss (hunk 1 above)
Comment (AshishDhama) [reply components/pricing_modal/pricing_modal.scss:301]: @julmondragon I will revert this we have a separate ticket for some colour issues related to this modal.

Comment (AshishDhama) [reply components/pricing_modal/pricing_modal.scss:301]: https://mattermost.atlassian.net/browse/MM-48570

Comment (julmondragon) [reply components/pricing_modal/pricing_modal.scss]: Thanks @AshishDhama !

**Review by hmhealey — APPROVED** [review]

**Comment by AshishDhama** [comment]
@mattermost/qa-core-team Please let me know how I can help test this.

**Comment by AshishDhama** [comment]
cc// @michaelgamble

**Review by furqanmlk — APPROVED** [review]

**Pull request merged**